### DISCUSSION TITLE
How can I change the invalid version message on sign-in?

### DISCUSSION THREAD

**Discussion by thelipe7** [Minestom/Minestom#1653] [Help]
This message:
![image](https://user-images.githubusercontent.com/101154133/210276099-32902059-7626-477b-921e-9904adeab5e5.png)

**Comment by ZakShearman** [comment]
You can't change it without modifying Minestom.

It's hardcoded here: https://github.com/Minestom/Minestom/blob/e5d0a43ba42c7023a93e8e33356909e614783ac7/src/main/java/net/minestom/server/network/packet/client/handshake/HandshakePacket.java#L37
